apps/web/app/api/user/me/route.ts

```diff
@@ -30,6 +30,17 @@ async function getUser({ userId }: { userId: string }) {
           pendingInvites: true,
         },
       },
+      members: {
+        select: {
+          organizationId: true,
+          role: true,
+          organization: {
+            select: {
+              name: true,
+            },
+          },
+        },
+      },
```

Comment:
in general, i'm not a fan of adding lots of joins to routes that are used across the app. if it's only for one-time usage.

but i guess here it's for the app layout, so we're basically always loading it. so maybe it's fine here, and slower to break it into 2 calls.